crates/hir_def/src/path.rs

```diff
@@ -44,6 +44,10 @@ pub enum ImportAlias {
 }
 
 impl ModPath {
+    pub fn from_src_unhygienic(path: ast::Path) -> Option<ModPath> {
```

Comment:
I need some way to turn the qualifier (`ast::NameRef`) into `a::mod::string` without any turbofishes ergo this.
If there's a better way, let me know.

Reply:
Yeah, I think that's not the best way to do this.

What we want is transforming an ast to a different AST/String. We are not interested in Name, hygiene and all that other semantic stuff. 

I think what we need here is just 

```rust
fn path_to_string_stripping_turbo_fish(path: &ast::Path) -> String {

}
```

which iterates the segments and assembles the string in a boring way. That's more code to write, but it's on the right abstraction layer (we could even put it to some utils module in `syntax`, where it belongs).